integ-tests/src/cucumberTest/java/cucumber/steps/helpers/InputCreator.java

```diff
@@ -20,95 +20,100 @@
 import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.CreateEnvironmentRequest;
 import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.DeleteEnvironmentRequest;
 import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.DescribeEnvironmentRequest;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.ListClustersRequest;
 import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.UpdateEnvironmentRequest;
-
+import java.util.Collections;
+import java.util.StringJoiner;
 import java.util.UUID;
+import lombok.Getter;
```

Comment:
Are we using Getter/Setter here?

Reply:
Should be on the `accountId` field, yep. Didn't want to apply it to the whole class.